README.md

```diff
@@ -16,9 +16,11 @@ The St. Olaf community, now in pocket size… rewritten in React Native.
 
 - [Install React Native](http://facebook.github.io/react-native/docs/getting-started.html#content)
 - [Install Yarn](https://yarnpkg.com/en/docs/install)
+- [Git Large File Storage (LFS)](https://help.github.com/articles/installing-git-large-file-storage/)
```

Comment:
"Install Git Large File Storage (LFS)"

I guess that link's fine. It does more hand-holding than anything else I've found.